tests/mlmodel_bedrock/test_chat_completion.py

```diff
@@ -98,11 +89,18 @@ def expected_events_no_convo_id(model_id):
 
 
 @reset_core_stats_engine()
-def test_bedrock_chat_completion_in_txn(exercise_model, expected_events):
+def test_bedrock_chat_completion_in_txn_with_convo_id(set_trace_info, exercise_model, expected_events):
     @validate_ml_events(expected_events)
     # One summary event, one user message, and one response message from the assistant
     @validate_ml_event_count(count=3)
-    @background_task()
+    # @validate_transaction_metrics(
```

Comment:
I assume you're leaving these here as a TODO once you merge the branches together.